cachi2/core/package_managers/npm.py

```diff
@@ -51,8 +51,8 @@ def __init__(self, name: str, path: str, package_dict: dict[str, Any]) -> None:
 
         :param name: the package name, which should correspond to the name in it's package.json
         :param path: the relative path to the package from the root project dir. This is set for
-                     for package-lock.json `packages` and falsy for `dependencies`.
-        :param package_dict: the raw dict for a package-lock.json `package` or `dependency`
+                     package-lock.json `packages`.
```

Comment:
nitpick: "This is set for package-lock.json `packages`" doesn't really make sense to me anymore (there's no other type of packages)

Reply:
removed